src/commands/edge_app.rs

```diff
@@ -333,6 +367,37 @@ impl EdgeAppCommand {
         Ok(())
     }
 
+    pub fn generate_mock_data(&self, path: &Path) -> Result<(), CommandError> {
+        let data = fs::read_to_string(path)?;
+        let manifest: EdgeAppManifest = serde_yaml::from_str(&data)?;
+        let edge_app_dir = path.parent().ok_or(CommandError::MissingField)?;
+
+        let default_metadata = Metadata::default();
+
+        let mut settings: HashMap<String, serde_yaml::Value> = HashMap::new();
+        for setting in &manifest.settings {
+            if setting.type_ != "secret" {
+                settings.insert(
+                    setting.title.clone(),
+                    serde_yaml::Value::String(setting.default_value.clone()),
```

Comment:
If default_values are used for settings - may be for dev server secrets can also use default_values?

Reply:
No, we don't want to store secret values in the file so users don't commit their secrets to git